packages/docs/docs/docs/writing_contracts.md

```diff
@@ -249,4 +249,25 @@ Then modifying `MyContract` by swapping the order in which the base contracts ar
 contract MyContract is B, A { }
 ```
 
-Violating any of these storage layout restrictions will cause the upgraded version of the contract to have its storage values mixed up, and can lead to critical errors in your application.
\ No newline at end of file
+You also cannot add new variables to base contracts, if the child has any variables of its own. Given the following scenario:
+```js
+contract Base {
+  uint256 base1;
+}
+
+contract Child {
+  uint256 child;
+}
+```
+
+And if `Base` is modified to add an extra variable:
+```js
+contract Base {
+  uint256 base1;
+  uint256 base2;
+}
+```
+
+Then `base2` whould be assigned the same slot as `child`. A workaround for this is to declare unused variables on base contracts that you may want to extend in the future, as a means of "reserving" those slots. Note that this trick does not involve increased gas usage.
```

Comment:
I think the first sentence may be confusing. I would change it to read "... the slot `child` where was stored in the previous version" or something like that.